build/build-report.js

```diff
@@ -96,8 +96,32 @@ async function buildEsModulesBundle() {
       rollupPlugins.commonjs(),
       // Exclude this 30kb from the devtools bundle for now.
       rollupPlugins.shim({
-        [`${LH_ROOT}/shared/localization/i18n-module.js`]:
-            'export const swapLocale = _ => {}; export const format = _ => {};',
+        [`${LH_ROOT}/shared/localization/i18n-module.js`]: `
+/**
+ * Returns a new LHR with all strings changed to the new requestedLocale.
+ * @param {LH.Result} lhr
+ * @param {LH.Locale} requestedLocale
+ * @return {{lhr: LH.Result, missingIcuMessageIds: string[]}}
+ */
+export function swapLocale(lhr, requestedLocale) {
+  return {
+    lhr,
+    missingIcuMessageIds: []
+  };
+};
+
```

Comment:
All this needs a comment... "only included for types" ??

Reply:
Also, perhaps we should do the type sed'ing stuff in LH repo, and check in the resultant .d.ts file (also asserting in CI it doesn't change). These types are gonna be a moving target.

Reply:
done